README.md

```diff
@@ -6,14 +6,53 @@
 $ bower install gaia-components/gaia-button
 ```
 
+Then include folowing files in HTML
+
+```html
+<link href="bower_components/gaia-icons/gaia-icons.css"></link>
+<script src="bower_components/gaia-list/gaia-button.js"></script>
+```
+
+You don't have to include `gaia-icon.css` if you do not use icon in your button.
+
+
 ## Examples
 
 - [Example](http://gaia-components.github.io/gaia-button/)
 
+
 ## Usage
 
+Button that contain text need to include `data-l10n-id` attribute for localization.
+
 ```html
-<gaia-button>normal button<gaia-button>
+<gaia-button><span data-l10n-id="normal_button"/><gaia-button>
+```
+
+### With icon
+
+* {Start icon} button text
+
+Button can show with icon at left
+
+```
+<button><span data-icon="back-light" data-l10n-id="back"/></button>
+```
+
+* button text {End icon}
+
+Button can show with icon at right
+
+```
+<button><span data-icon-end="forward-light" data-l10n-id="forward"/></button>
+```
+
+* {Start icon} button text {End icon}
+
+Button can show with icon on both side
+
+```
+<button><span data-icon="back-light" data-icon-end="forward-light" data-l10n-id="quote">Quote</span></button>
```

Comment:
I think the most error-proof way is as above, to add data-l10n-id to the top level container describing icons and the button text altogether. Since we have button semantics, screen reader will only pick up aria-label and ignore inner content.